### PR TITLE
Dev/robin/9530 end to end scitt

### PR DESCRIPTION
**Pull request by robinbryce**
Update scitt samples to show end to end scitt statement creation, registration and verification

Implements a simple common configuration context pattern for interacting with the DataTrails SCITT and proprietary apis and consistent use of logging.
The pattern accomodates CI config via environment and imperative config for command line use.

Also, refactor the scripts and packages so they can be consumed by scitt-actions and the like.

### CODE REVIEW

File: scitt/statement_creation.py [hunk 1]
```diff
@@ -0,0 +1,175 @@
+"""The issuer creates the statement and signs it.
```
Comment (robinbryce) [inline scitt/statement_creation.py:1]: Will restore this soon. The registration script servces as a full example of receipt verification for now

**Comment by SteveLasker** [comment]
Updated to:
- support cose-hash-envelop
- meta-map

I got all the tests running locally
- `task test`
- realtime tests work as the functions pass for create/register a signed/hashed statement
- ran the updated (draft) doc update at: https://github.com/datatrails/datatrails-docs/pull/934

It is not passing CI, 😢 and I could use a wee bit of help

**Review by SteveLasker — DISMISSED** [review]
LGTM, 
We need to resolve #25, however that will be a separate coordinated backend PR.

**Review by SteveLasker — DISMISSED** [review]
LGTM

**Review by SteveLasker — APPROVED** [review]
LGTM

**Review by JAG-UK — APPROVED** [review]
Big improvement

**Review by JAG-UK — APPROVED** [review]
Again

**Pull request merged**